src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -673,6 +673,11 @@ void SizeZoneEquipment(EnergyPlusData &state)
                 state.dataSize->CalcZoneSizing(state.dataSize->CurOverallSimDay, ControlledZoneNum).CoolTstatTemp =
                     state.dataHeatBalFanSys->ZoneThermostatSetPointHi(ActualZoneNum);
             }
+        } else {
+            state.dataSize->CalcZoneSizing(state.dataSize->CurOverallSimDay, ControlledZoneNum).HeatTstatTemp =
+                state.dataHeatBalFanSys->ZoneThermostatSetPointLo(ActualZoneNum);
+            state.dataSize->CalcZoneSizing(state.dataSize->CurOverallSimDay, ControlledZoneNum).CoolTstatTemp =
+                state.dataHeatBalFanSys->ZoneThermostatSetPointHi(ActualZoneNum);
```

Comment:
@matthew-larson I am wondering why you did not set the following variable under the *else* condition? This king of changes may lead to unexplained differences (I see big cooling system sizing diffs). The previous code (before the change) is not the same as the current one. Unless there is good reason not to do so, I prefer to maintain the original logic, i.e. set  .CoolZoneRetTemp variable under the *else* logic as well.


`state.dataSize->CalcZoneSizing(state.dataSize->CurOverallSimDay, ControlledZoneNum).CoolZoneRetTemp = RetTemp;
`

Also in the *else" logic you do not check whether the TempZoneThermostatSetPoint() value is greater than 0. Any reason for not doing it?

Reply:
I believe this is where many of the diffs came from, not setting the .`CoolZoneRetTemp` and `.HeatZoneRetTemp` in the `else` logic.  I added this to the `else` logic and this seemed to reduce the number of diffs significantly.  Need to investigate the remaining diffs however.

The reason I didn't check the `TempZoneThermostatSetPoint()` value is that the value of `.HeatTstatTemp` and `.CoolTstatTemp` would be set to `ZoneThermostatSetPointLo` and `ZoneThermostatSetPointHi` respectively no matter the value of `TempZoneThermostatSetPoint()`.